docs/release.rst

```diff
@@ -320,6 +320,9 @@ in the Fedora version specific branch on Anaconda side.
 Commit the result. The commit will become one of the few exclusive release branch commits,
 as we can't let it be merged back to master via the devel branch for obvious reasons.
 
+After doing this please verify that Pykickstart support that Fedora version and version
+of the current Fedora Rawhide (means *<version> + 1*) and if not please file
+an `issue <https://github.com/pykickstart/pykickstart/issues>`_ on the pykickstart project.
```

Comment:
There are `Pykickstart` and `pykickstart` in the paragraph. It is not consistent.

Reply:
Fixed.